Tests/AGSnapshotHelperTests/FBSnapshotTestCase+AGSnapshotHelperTests.m

```diff
@@ -0,0 +1,56 @@
+//
+//  FBSnapshotTestCase+AGSnapshotHelperTests.m
+//  AGSnapshotHelper
+//
+//  Created by Adam Grzegorowski on 15/01/16.
+//  Copyright © 2016 allegro. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import <FBSnapshotTestCase/FBSnapshotTestController.h>
+#import <FBSnapshotTestCase/FBSnapshotTestCase.h>
+#import <AGSnapshotHelper/FBSnapshotTestCase+AGSnapshotHelper.h>
+
+static CGRect AGViewFrame = (CGRect){{0.0f, 0.0f}, {50.0f, 50.0f}};
+
+@interface FBSnapshotTestCase_AGSnapshotHelperTests : FBSnapshotTestCase
+@end
+
+@implementation FBSnapshotTestCase_AGSnapshotHelperTests
+
+- (void)testTestViewOfContentSizeCategoryWithDisabledRecodedModeShouldCompareView {
```

Comment:
As you wrote below this test doesn't have any object which could react on content size category changes, so I remove `OfContentSizeCategory` part from test names.